multi-node/generic/worker-install.sh

```diff
@@ -150,6 +209,8 @@ kind: Pod
 metadata:
   name: kube-proxy
   namespace: kube-system
+  annotations:
```

Comment:
Lets file an issue instead. I think I made a similar one for the nsenter hack too.